lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -124,6 +141,19 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
     );
   }
 
+  Path _generatePath(double centerX, double centerY, double radius, int count) {
```

Comment:
I think we can rename it to `_generatePolygonPath` to make it more descriptive.
What do you think?